wtxmgr/tx.go

```diff
@@ -375,7 +375,7 @@ func (s *Store) insertMinedTx(ns walletdb.ReadWriteBucket, rec *TxRecord,
 	// If this transaction previously existed within the store as
 	// unconfirmed, we'll need to move it to the confirmed bucket.
 	if v := existsRawUnmined(ns, rec.Hash[:]); v != nil {
-		return s.moveMinedTx(ns, rec, block)
+		s.moveMinedTx(ns, rec, block)
```

Comment:
The error here is now being ignored... 

The comment above could also be updated to reflect that we don't terminate here to ensure that any unmined conflicting transactions are properly removed.

Reply:
Fixed.